### ISSUE TITLE
arm64 shows as arm64/undefined

### ISSUE THREAD

**Issue by tonistiigi** [estesp/mquery#6]
Based on containerd/platform implementation, variant should not be required for arm64 and missing in the normalized mode. But in mquery output it shows as `arm64/undefined` if missing.

```
> docker run --rm mplatform/mquery tonistiigi/copy:v0.1.5
Image: tonistiigi/copy:v0.1.5
 * Manifest List: Yes
 * Supported platforms:
   - linux/amd64
   - linux/arm64/undefined
   - linux/arm/v7
   - linux/ppc64le
   - linux/s390x
```

```
» go run dhub.go manifest tonistiigi/copy v0.1.5                                                               !10266
{
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "schemaVersion": 2,
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:622cb58662caba19119d8470e5cb355b08f2a94a4d30996108890f74197ffa54",
         "size": 1365,
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:8a0ad02b8fa9a97bf472ac1779447d227b3369e22c67d22b742df805bd331ea3",
         "size": 1155,
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:f05dc1efac3e9a2f2b71af1a66c87f0e2c1072e6cadce9d405b54a1204e7ea4c",
         "size": 1155,
         "platform": {
            "architecture": "arm",
            "os": "linux",
            "variant": "v7"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:2b09703b0299eea011cec989798bc2d107008417abe1ecc23484a7ea30e8b514",
         "size": 1155,
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:e197a4f3565a1ebec0f23258c3fc51d900d7cf982742d2ea9b1b9cf946fa7bd9",
         "size": 1155,
         "platform": {
            "architecture": "s390x",
            "os": "linux"
         }
      }
   ]
}
```

**Comment by estesp** [comment]
Thanks for the report; fixed in #7 and tested with this output:
```
docker run --rm mplatform/mquery tonistiigi/copy:v0.1.5
Image: tonistiigi/copy:v0.1.5
 * Manifest List: Yes
 * Supported platforms:
   - linux/amd64
   - linux/arm64
   - linux/arm/v7
   - linux/ppc64le
   - linux/s390x
```

**Comment by tonistiigi** [comment]
@estesp Thanks!
